parseutils.c

```diff
@@ -542,6 +542,8 @@ parse_keqv_line(char *line)
 	return values;
 }
 
+
```

Comment:
gratuitous white-space addition

Reply:
ya, will deal with this, thanks for reviewing